index.test.js

```diff
@@ -0,0 +1,74 @@
+const http = require('http');
+const supertest = require('supertest');
+const handler = require('./src/handler');
+const parseEnv = require('./src/parse-env');
+
+const describeIfEnv = (process.env.AIRTABLE_API_KEY && process.env.AIRTABLE_BASE_ID) ? describe : xdescribe;
+
+const getExistingPosts = async () => {
+  const config = parseEnv(process.env);
+  const server = http.createServer(handler(config));
+
+  return (await supertest(server).get('/v0/Posts')).body;
+};
+
+describeIfEnv('integration', () => {
+  this.config = parseEnv(process.env);
```

Comment:
Lets use the JS API, not `parseEnv`.

We've already got tests for that `parseEnv`, and since we don't want to inherit and permissions or other configuration from the environment, we should just extract the two environment variables we care about (`AIRTABLE_BASE_ID` and `AIRTABLE_API_KEY`):

```js
const baseConfig = {
  airtableBaseId: process.env.AIRTABLE_BASE_ID,
  airtableApiKey: process.env.AIRTABLE_API_KEY,
};

describe('micro-airtable-api', () => { /* ... */ });
```

You can then "mixin" specific configurations you need like so:

```js
describe('allowedMethods "*"', () => {
  const config = { ...baseConfig, allowedMethods: '*' };
});
```